cfme/tests/ssui/test_ssui_service_dialog.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+import fauxfactory
+import pytest
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.appliance import ViaSSUI
+from cfme.utils.appliance.implementations.ssui import navigate_to
+
+
+@pytest.fixture(scope="function")
+def tagcontrol_dialog(appliance):
+    service_dialog = appliance.collections.service_dialogs
+    dialog = "dialog_{}".format(fauxfactory.gen_alphanumeric())
+    element_data = {
+        'element_information': {
+            'ele_label': "Service Level",
+            'ele_name': "service_level",
+            'ele_desc': "service_level_desc",
+            'choose_type': "Tag Control",
+        },
+        'options': {
+            'field_category': "Service Level",
+            'field_required': True
+        }
+    }
+    sd = service_dialog.create(label=dialog, description="my dialog")
+    tab = sd.tabs.create(tab_label='tab_{}'.format(fauxfactory.gen_alphanumeric()),
+        tab_desc="my tab desc")
+    box = tab.boxes.create(box_label='box_{}'.format(fauxfactory.gen_alphanumeric()),
+        box_desc="my box desc")
+    box.elements.create(element_data=[element_data])
+    yield sd
+
+
+@pytest.fixture(scope="function")
+def catalog(appliance):
+    catalog = "cat_" + fauxfactory.gen_alphanumeric()
+    cat = appliance.collections.catalogs.create(name=catalog, description="my catalog")
+    yield cat
+
+
+@pytest.fixture(scope="function")
+def catalog_item(appliance, tagcontrol_dialog, catalog):
+    item_name = fauxfactory.gen_alphanumeric()
+    catalog_item = appliance.collections.catalog_items.create(
+        appliance.collections.catalog_items.GENERIC,
+        name=item_name,
+        description="my catalog", display_in=True, catalog=catalog,
+        dialog=tagcontrol_dialog)
+    return catalog_item
+
+
+@pytest.mark.parametrize('context', [ViaSSUI])
+def test_tagdialog_catalog_item_ssui(appliance, context, catalog_item):
```

Comment:
Your context already pointing its ssui test. I would give better name. I would add `_` in between `tag` and `dialog`

Reply:
Question:
If its part of `ssui`. As per my understanding its specific for `ssui` then why you are parametrizing? Second though are you going to add `ViaUI` as well?

Reply:
@digitronik yes this is ssui test for that we have to add context for ssui login.In this test I will test only ViaSSUI.

Reply:
Made changes